runtime.ts

```diff
@@ -195,7 +195,8 @@ class Compiler {
     inlineSourceMap: true,
     lib: ["es2017"],
     inlineSources: true,
-    target: ts.ScriptTarget.ES2017
+    target: ts.ScriptTarget.ES2017,
+    esModuleInterop: true
```

Comment:
Why is this needed?

Reply:
Oh left that from a previous experiment, not required